XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs

```diff
@@ -1,11 +1,11 @@
-﻿using XamarinCommunityToolkit.Converters;
+﻿using Microsoft.Toolkit.Xamarin.Forms.Converters;
```

Comment:
Do we care about namespace ordering? If so log an issue and I could fix with R#

Reply:
Usually, I have the habit to order them. In my last two PRs, I wanted to order them from longer to shorten ones. 😊 Not sure if there is the policy on that for the entire project. But I am ok with idea to have them ordered. 😊

Reply:
Typically it's all System namespaces and then other namespaces, ordered alphabetically